src/dc/components/accommodation-blocks/hotel-venue-distance.js

```diff
@@ -10,7 +10,7 @@ export const HotelVenueDistance = ({hotelLatitude, hotelLongitude}) => {
         return (<></>);
     }
 
-    let venuePoint = new Geopoint(Number(venueConfig.destinationLocation.latitude), Number(venueConfig.destinationLocation.longitude));
+    let venuePoint = new Geopoint(Number(venueConfig.destinationCity.latitude), Number(venueConfig.destinationCity.longitude));
```

Comment:
Don't agree with this change, the location of the venue is more precise than the city. I used the exact location of the event in this variable.

Reply:
fixed